storagenode/storagenodedb/infodb.go

```diff
@@ -269,6 +272,31 @@ func (db *InfoDB) Migration() *migrate.Migration {
 					`CREATE INDEX idx_pieceinfo_expiration ON pieceinfo(piece_expiration) WHERE piece_expiration IS NOT NULL`,
 				},
 			},
+			{
+				Description: "Clear Tables from Alpha data",
+				Version:     11,
+				Action: migrate.SQL{
+					`DELETE FROM pieceinfo`,
+					`DELETE FROM used_serial`,
+					`DELETE FROM order_archive`,
+				},
+			},
+			{
+				Description: "Free Storagenodes from trash data",
+				Version:     12,
+				Action: migrate.Func(func(log *zap.Logger, mgdb migrate.DB, tx *sql.Tx) error {
+					deletecmd := exec.Command("rm", "-r",
```

Comment:
I think Windows doesn't have `rm` command by default.

Reply:
The Docker container all have that. Thats why we can use it.

Reply:
I think we can also just call `os.RemoveAll`

Reply:
RemoveAll on the specific folder?

Reply:
Yeah, instead of `rm -r <dir>`, you can do `os.RemoveAll(<dir>)`.